src/service/MxGraphFactoryService.ts

```diff
@@ -0,0 +1,27 @@
+import { mxgraphFactory } from 'ts-mxgraph';
+
+const { mxClient, mxUtils, mxGraph, mxGraphModel, mxConstants, mxPerimeter, mxPoint } = mxgraphFactory({
```

Comment:
:question: could we do the call in the factory method instead? 
This will avoid global state, in addition the values are already stored in the singleton so should not be needed outside
In addition, this allows extra configuration prior calling the singleton. Currently, the mxgraphfactory function is called on the file import